runtime/dolphin/src/lib.rs

```diff
@@ -398,6 +399,17 @@ impl pallet_sudo::Config for Runtime {
     type RuntimeCall = RuntimeCall;
 }
 
+parameter_types! {
+    pub const NameServicePalletId: PalletId = NAME_SERVICE_PALLET_ID;
+}
+
+impl pallet_name_service::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type PalletId = NameServicePalletId;
+    type RegisterWaitingPeriod = ConstU32<2>;
+    //type WeightInfo = weights::pallet_name_service::SubstrateWeight<Runtime>;
```

Comment:
todo: weights
you just need to run the all-benchmarks workflow on this branch with calamari-dev